support_branding/static/src/js/support_branding.js

```diff
@@ -0,0 +1,170 @@
+                    {
+                        self[record.key.replace('.', '_')] = record.value;
+                    });
+                });
+        },
+    });
+
+    instance.web.CrashManager.include({
+        show_error: function(error)
+        {
+            this._super.apply(this, arguments);
+            jQuery('.support-branding-submit-form').each(function()
+            {
+                var $form = jQuery(this),
+                    $button = $form.find('button'),
+                    $description = $form.find('textarea[name="description"]'),
+                    $subject = $form.find('input[name="subject"]'),
+                    $body = $form.find('input[name="body"]');
+                if(instance.client.session.support_branding_support_email)
+                {
+                    $form.attr(
+                        'action',
+                        'mailto:' + instance.client.session.support_branding_support_email);
+                    $form.parents('.ui-dialog').find('.ui-dialog-content')
+                        .css('max-height', '70vh');
+                    $button.click(function(ev)
+                    {
+                        var mail_mail = new instance.web.Model('mail.mail');
+                        if(!$description.val())
+                        {
+                            $description.parent().addClass('oe_form_invalid');
+                            ev.preventDefault();
+                            return;
+                        }
+                        mail_mail.call(
+                            'create',
+                            [{
+                                state: 'outgoing',
+                                auto_delete: false,
+                                email_to: instance.client.session.support_branding_support_email,
+                                subject: $subject.val(),
+                                body_html: jQuery('<div/>').append(
+                                    jQuery('<div/>').text($description.val()),
+                                    jQuery('<pre/>').text($body.val())
+                                ).html(),
+                            }])
+                        .then(function(mail_id)
+                        {
+                            return mail_mail.call('send', [[mail_id]]);
+                        }, function()
+                        {
+                            // if the call failed, fire the mailto link
+                            // hoping there is a properly configured email
+                            // client
+                            $body.val($description.val() + '\n' + $body.val())
+                            $button.unbind('click');
+                            $button.click();
+                        })
+                        .then(function()
+                        {
+                            //TODO close the dialog
+                            $form.parents('.ui-dialog').hide();
```

Comment:
Upstream does that: https://github.com/odoo/odoo/blob/7.0/addons/web/static/src/js/chrome.js#L289

So I think in this case, you need to save the original `this` and call `$(self).dialog("close");`